functions/src/firebase/env.ts

```diff
@@ -30,3 +33,42 @@ export const getTwilioEnv = (): TwilioEnv => {
         apiKeySecret: twilio.apikeysecret,
     }
 }
+
+export const getAppEnv = (): AppEnv => {
+    const { app } = functions.config()
+    if (!app.domain || !app.emailfrom) {
+        throw new InternalServerError('Missing app domain or emailfrom env')
+    }
+    return {
+        domain: app.domain,
+        emailFrom: app.emailfrom,
+    }
+}
+
+export const getOVHEnv = (): OVHCredentials => {
+    const { ovh } = functions.config()
+    if (!ovh.consumerkey || !ovh.servicename || !ovh.appsecret || !ovh.appkey) {
+        throw new InternalServerError(
+            'Missing ovh consumerkey or servicename or appsecret or appkey env'
+        )
+    }
+    return {
+        consumerkey: ovh.consumerkey,
+        servicename: ovh.servicename,
+        appsecret: ovh.appsecret,
+        appkey: ovh.appkey,
```

Comment:
as sayed in another PR, this are env declared on the servers, they where all lowercase since the beginning of the project and impact the production as well as the dev.